src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,25 +33,33 @@
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              Todo #{todo.id}
             </div>
 
             {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-            <button type="button" className="delete" data-cy="modal-close" />
+            <button
+              onClick={() => changeModal(-1)}
```

Comment:
Consider using a more descriptive value than `-1` when calling `changeModal` to indicate closing the modal. Using a specific constant or `null` can make the code more readable and self-explanatory.